controllers/constants/constants.go

```diff
@@ -42,9 +42,10 @@ const (
 	MonitoringServiceAccount = "prometheus-k8s"
 
 	// Loki roles
-	LokiCRWriter  = "netobserv-writer"
-	LokiCRBWriter = "netobserv-writer-flp"
-	LokiCRReader  = "netobserv-reader"
+	LokiCRWriter        = "netobserv-writer"
+	LokiCRBWriter       = "netobserv-writer-flp"
+	LokiCRReader        = "netobserv-reader"
+	MaxOvsMonitorEvents = 4
```

Comment:
I'm not sure to get the impact of this yet. What would be the advantage to get more / less events metadatas ?

Reply:
for the same flow we can have multiple events md if we wanted to trace different feature for example we wanted to trace ACL, NAT, QoS for example we will see array populated with 4 metadata strings each for one of the above features, while if we trace ACL only for example then only one event md will be populated

Reply:
ok makes sense. So are we going to expose which features to trace in the FlowCollector CR ?

If so we could adapt the console plugin config dynamically and bind each field with its proper name / type etc